gammapy/irf/tests/test_effective_area.py

```diff
@@ -57,3 +59,72 @@ def test_EffectiveAreaTable_write():
         # see e.g. https://travis-ci.org/gammapy/gammapy/jobs/31056341#L1162
         # assert hdu_list[1].verify_checksum() == 1
         assert len(hdu_list) == 2
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+def test_OffsetDependentEffectiveAreaTable():
+    # Read test effective area file
+    effarea = OffsetDependentEffectiveAreaTable.from_fits(
+        load_aeff2D_fits_table())
+
+    interpolation_methods = ('linear', 'spline')
+
+    for method in interpolation_methods:
```

Comment:
Can you please use pytest parametrize instead of looping in the test function?

Docs: https://pytest.org/latest/parametrize.html
Examples: https://github.com/astropy/astropy/search?utf8=%E2%9C%93&q=parametrize

It's a bit complicated to learn how to use this, but pretty powerful and worth it in the long run.